cmd/kritis/kubectl/plugins/resolve/cmd/root.go

```diff
@@ -75,3 +92,38 @@ func resolveFilepaths() error {
 	}
 	return nil
 }
+
+func outputResults(substitutes map[string]string, writer io.Writer) error {
+	if apply {
+		return applyChanges(substitutes, writer)
+	}
+	print(substitutes, writer)
+	return nil
+}
+
+// prints the final replaced kubernetes manifest to given writer
+func print(substitutes map[string]string, writer io.Writer) {
+	for file, contents := range substitutes {
+		fmt.Fprintln(writer, fmt.Sprintf("---%s---", file))
+		fmt.Fprintf(writer, contents)
+		fmt.Fprintln(writer)
+	}
+}
+
+func applyChanges(substitutes map[string]string, writer io.Writer) error {
+	kubectl := os.Getenv("KUBECTL_PLUGINS_CALLER")
```

Comment:
Is there a reason we need to do this? Can you add some comments here?

Reply:
This gives us the full path to the kubectl binary, which I figured would be better to use if we could get it. Otherwise, we assume it's on the PATH